daml/Marketplace/ExchangeParticipant.daml

```diff
@@ -18,10 +18,18 @@ template ExchangeParticipantInvitation
 
     controller exchParticipant can
       ExchangeParticipantInvitation_Accept : ContractId ExchangeParticipant
-        do create ExchangeParticipant with ..
+        do
+          create DismissibleNotification with
+            sender = exchParticipant, senderRole = None
```

Comment:
Currently the Exchange invitations are only for Investors in the UI, but it looks like that will change to also allow brokers to be invited, so I left this as `None` for now.

Once that change happens in the UI, I'd like to consider updating `ExchangeParticipantInvitation_Accept/Reject` to have an `Optional MarketRole` field that can be filled in when selecting the party for invitation (which can have knowledge of the role, similar to how allocate to provider works now). Currently it would only have an effect on the `DismissibleNotification`, but that information could also be useful elsewhere.